pkg/columns/sort/sort_test.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sort
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/kinvolk/inspektor-gadget/pkg/columns"
+)
+
+func TestSorter(t *testing.T) {
+	type testData struct {
+		Int     int     `column:"int"`
+		Uint    uint    `column:"uint"`
+		String  string  `column:"string"`
+		Float32 float32 `column:"float32"`
+		Float64 float64 `column:"float64"`
+		Bool    bool    `column:"bool"`
+	}
+	testEntries := []*testData{
```

Comment:
Is it possible to sort by a field that is in an embedded struct? Same test you did for `TestGroupSum`.

Reply:
Yes, it's possible. I added a test for that as well 👍